server/repository/src/db_diesel/temperature_log.rs

```diff
@@ -58,7 +58,7 @@ impl<'a> TemperatureLogRepository<'a> {
     }
 
     pub fn count(&self, filter: Option<TemperatureLogFilter>) -> Result<i64, RepositoryError> {
```

Comment:
Change in this file is just moving create_filtered_query into the repository struct